tortoise/backends/base/config_generator.py

```diff
@@ -88,17 +93,21 @@ def expand_db_url(db_url: str, testing: bool = False) -> dict:
     vmap.update(db["vmap"])
     params[vmap["path"]] = path
     if vmap.get("hostname"):
-        params[vmap["hostname"]] = str(url.hostname or "")
+        params[vmap["hostname"]] = None if not url.hostname else str(url.hostname)
```

Comment:
Is url.hostname not none if not defined? Or you could do `url.hostname or None`
This way seems a bit verbose?

Reply:
Thank. I've just made code less verbose. No need to do `str(url.username)` because `url.username` won't be other type than string and `None`.